src/shared/components/ChannelPreviewBase/ChannelPreviewBase.tsx

```diff
@@ -74,7 +74,7 @@ export const ChannelPreviewBase: React.FC<ChannelPreviewBaseProps> = ({
                       timeout={parseInt(transitions.timings.loading) * 0.5}
                       classNames={transitions.names.fade}
                     >
-                      <VideoCount variant="subtitle2">{videoCount ? `${videoCount} Uploads` : '⠀'}</VideoCount>
+                      <VideoCount variant="subtitle2">{`${videoCount ?? 0} Uploads`}</VideoCount>
```

Comment:
We resolve video count asynchronously so we shouldn't fallback to 0. This will lead to preview showing "0 uploads" and then changing to "X uploads" once loading is finished

Reply:
I don't think its the case cause the component transitions in when videoCount is truthy but we also don't need the 0 anyways